### DISCUSSION TITLE
Home Assistant Energy Monitor

### DISCUSSION THREAD

**Discussion by RoySalisbury** [syssi/esphome-jk-bms#267] [Q&A]
The HA Energy Monitor dashboard has the ability to add in battery in/out totals.  How can I get the JK BMS entity values to be used for these (Charging Power/Discharging Power).  I think it has to due with some meta data for sensor type.

I have 5 BMS's so eventually will use a helper to create a new aggregate (sum) sensor and use that, but to start with, just adding one bms would be a start.

**Comment by syssi** [comment]
Please merge the important/new parts of this YAML into your configuration: https://github.com/syssi/esphome-jk-bms/blob/main/yaml-snippets/esp32-ble-energy-dashboard.yaml

**Reply by RoySalisbury** [reply]
I have added this to all my ESP32 BMS devices.  I can see it found the entities in HA, and am just now waiting for it to show up in the energy dashboard (I added all the BMS's).

HA never stops amazing me on how configurable it is. :)
